src/beacons/use-is-beacons-enabled.tsx

```diff
@@ -0,0 +1,13 @@
+import {useRemoteConfig} from '@atb/RemoteConfigContext';
+import {useDebugOverride} from '@atb/debug';
+import {StorageModelKeysEnum} from '@atb/storage';
+
+export const useIsBeaconsEnabled = () => {
+  const {enable_beacons} = useRemoteConfig();
+  const [debugOverride] = useBeaconsEnabledDebugOverride();
```

Comment:
If useIsBeaconsEnabled is going to be used reactively, it is fine as is.
However if useIsBeaconsEnabled is going to be used as a one-time app start init call for beacon activation, you also want to retrieve and return debugOverrideReady (see example in the useFlexibleTransportEnabled hook). This is because until debugOverrideReady is true, enable_beacons will be set to its default value, not a value read from storage.

Reply:
Good point! I'll create a new pr for this

Reply:
https://github.com/AtB-AS/mittatb-app/pull/3884